temba/msgs/management/commands/recalc_msg_label_count.py

```diff
@@ -0,0 +1,92 @@
+from celery import shared_task
+from django.core.management import BaseCommand
+from django.db.models import Count, Case, When, IntegerField, Q
+
+from temba.msgs.models import Msg, OUTGOING, INCOMING, WIRED, SENT, DELIVERED, IVR, FLOW, SystemLabelCount, SystemLabel
+from temba.orgs.models import Org
+
+
+@shared_task(track_started=True, name="get_calculated_values")
+def get_calculated_values(org_id):  # pragma: no cover
+    label_mapping = dict(
+        text_flows=SystemLabel.TYPE_FLOWS,
+        voice_flows=SystemLabel.TYPE_FLOW_VOICE,
+        sent_text=SystemLabel.TYPE_SENT,
+        sent_voice=SystemLabel.TYPE_SENT_VOICE,
+    )
+
+    results = Msg.objects.filter(org=org_id).aggregate(
+        text_flows=Count(
+            Case(
+                When(direction=INCOMING, visibility=Msg.VISIBILITY_VISIBLE, msg_type=FLOW, then=1),
+                output_field=IntegerField(),
+            )
+        ),
+        voice_flows=Count(
+            Case(
+                When(direction=INCOMING, visibility=Msg.VISIBILITY_VISIBLE, msg_type=IVR, then=1),
+                output_field=IntegerField(),
+            )
+        ),
+        sent_voice=Count(
+            Case(
+                When(
+                    direction=OUTGOING,
+                    visibility=Msg.VISIBILITY_VISIBLE,
+                    status__in=(WIRED, SENT, DELIVERED),
+                    msg_type=IVR,
+                    then=1,
+                ),
+                output_field=IntegerField(),
+            )
+        ),
+        sent_text=Count(
+            Case(
+                When(
+                    Q(direction=OUTGOING)
+                    & Q(visibility=Msg.VISIBILITY_VISIBLE)
+                    & Q(status__in=(WIRED, SENT, DELIVERED))
+                    & ~Q(msg_type=IVR),
+                    then=1,
+                ),
+                output_field=IntegerField(),
+            )
+        ),
+    )
+
+    for key, count in results.items():
+        label_type = label_mapping[key]
```

Comment:
Let's add some logs, that way we can follow what's happening with the process. E.g. something like "Recalculating for Org 1, label type S"